android/src/main/java/com/razorpay/razorpay_flutter/RazorpayFlutterPlugin.java

```diff
@@ -23,6 +23,9 @@ public class RazorpayFlutterPlugin implements FlutterPlugin, MethodCallHandler,
     private RazorpayDelegate razorpayDelegate;
     private ActivityPluginBinding pluginBinding;
     private static String CHANNEL_NAME = "razorpay_flutter";
+    Map<String, Object> _arguments;
```

Comment:
@shibu000 any reason for this variable? Can we look into inlining the requirement in the switch condition itself?

Reply:
@vivekshindhe This variable is used for holding parameter value which we are getting from dart to the native side, As we are using this variable multiple places inside the function, we thought of avoiding inlining initialization.